challenge_1/go/tml/src/strrev.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+    "fmt"
+    "os"
+    "bufio"
+)
+
+func strrev(s string) string {
```

Comment:
You should capitalize methods you want exported from the package, as this makes it public. Starting an identifier with lower-case makes it package private.